SQF/dayz_code/Configs/CfgVehicles/Car/HMMWV.hpp

```diff
@@ -1079,3 +1081,27 @@ class HMMWV_M1151_M2_CZ_DES_EP1_DZE: HMMWV_M1151_M2_DES_Base_EP1_DZE
 		mat[] = {"Ca\wheeled_E\HMMWV\data\hmmwv_body_1.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_body_1_damage.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_body_1_destruct.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_hood.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_hood_damage.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_hood_destruct.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_parts_1.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_parts_1_damage.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_parts_1_destruct.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_regular_1.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_regular_1_damage.rvmat","Ca\wheeled_E\HMMWV\data\hmmwv_regular_1_destruct.rvmat","Ca\wheeled_E\HMMWV\Data\hmmwv_gpk_tower.rvmat","Ca\wheeled_E\HMMWV\Data\hmmwv_gpk_tower_damage.rvmat","Ca\wheeled_E\HMMWV\Data\hmmwv_gpk_tower_destruct.rvmat","Ca\Ca_E\data\default.rvmat","Ca\Ca_E\data\default.rvmat","Ca\Ca_E\data\default_destruct.rvmat"};
 	};
 };
+
+class HMMWV_Armored;
+class HMMWV_Armored_DZ: HMMWV_Armored {
+	scope = public;
+	displayname = "HMMWV (M240) Woodland";
+	maxspeed = 100;
+	transportMaxWeapons = 4;
+	transportMaxMagazines = 100;
+	transportmaxbackpacks = 4;
+	armor = 80;
+	damageResistance = 0.00581;
+};
+
+class HMMWV_M2;
+class HMMWV_M2_DZ: HMMWV_M2 {
+	scope = public;
+	displayname = "HMMWV (M2) Woodland";
+	maxspeed = 100;
+	transportMaxWeapons = 4;
+	transportMaxMagazines = 100;
+	transportmaxbackpacks = 4;
```

Comment:
The consensus was to leave the cargo spaces alone for these types of vehicles. All the regular landrover, hmmwv, vans, SUV, pickup, vodnik, etc. have 10 weapons, 50 mags and 2 backpacks (vodnik has 4 backpacks). You should leave these out for these humvees and they will inherit their default which is 10/50/5.